lib/clean.js

```diff
@@ -1,19 +1,15 @@
 'use strict';
 
-var merge = require('lodash').merge;
-var del = require('del');
+import {merge} from 'lodash';
+import del from 'del';
 
-var defaults = {
+const defaults = {
   dest: 'dist'
 };
 
-module.exports = function (gulp, options) {
+module.exports = (gulp, options) => {
   var opts = merge({}, defaults, options);
   var dest = opts.dest;
 
-  gulp.task('clean', function (done) {
-    del(dest).then(function () {
-      done();
-    });
-  });
+  gulp.task('clean', done => del(dest).then(() => done()));
```

Comment:
> Note I also think you should be able to pass the done function reference without wrapping it in a callback and invoking it :).

I should have probably **commented** the purpose for this "fix", but it appears that this was originally attempted with unsuccessful results: https://github.com/cloudfour/core-gulp-tasks/pull/4

I agree that it doesn't make sense, and perhaps needs further investigation.

Looking at this (https://github.com/sindresorhus/del#usage), it appears that the callback is invoked with an array of deleted paths as the first argument. I wonder if that's an issue.